sdks/python/apache_beam/coders/coders.py

```diff
@@ -216,6 +222,9 @@ def __eq__(self, other):
             and self._dict_without_impl() == other._dict_without_impl())
     # pylint: enable=protected-access
 
+  def __hash__(self):
+    return hash(type(self))
```

Comment:
Any particular reason for this change?  Previously, the hash would default to `object.__hash__`, which tries to give a different hash code for each instance.  This change would give the same hash code for each class, which may not be desirable, since coders in general could be parameterized.

Reply:
You're right. I've changed the hash to match the `__eq__` check.